cmd/eval.go

```diff
@@ -62,17 +62,28 @@ Example: combo eval -r REPLACE_ME=1,2,3 path/to/file
 				return err
 			}
 
-			templateFile, err := os.Open(args[FilePathArgsIndex])
+			// Determine if input is from pipe or designated input file
+			fi, err := os.Stdin.Stat()
 			if err != nil {
-				return fmt.Errorf("failed to read file specified: %w", err)
+				return fmt.Errorf("error accessing STDIN")
```

Comment:
It is normally best practice to include the error that caused this to occur. You can do this with by including a %w in the string and then the error as a second parameter (similar to what is seen in the diff).